dissemin/settings/common.py

```diff
@@ -181,7 +183,6 @@
                     "django.template.context_processors.tz",
                     "django.template.context_processors.request"
                 ),
-                'debug': True
```

Comment:
Right, that was useless.

Also, is it fine to turn off template caching in dev, or should I put the commit aside?